.github/workflows/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

Comment:
Is this necessary? Any meaningful PR creation must either follow or precede some kind of push... I think?

Reply:
If I understand [this](https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2) this block will run actions on a push to `master` OR a PR created with a target of `master`.  So feature branches won't have actions run until they turn into a PR.

On rethinking this, I think we want this to trigger on any push (and thus any PR)?  Otherwise you could be working on a feature branch and only find out after creating your PR that something is broken according to CI.

Reply:
Based on [this](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request) I've removed the push filter and all of the PRs section in f18cb85.

Reply:
:+1: